segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/PravegaRequestProcessor.java

```diff
@@ -973,6 +973,11 @@ private Void handleException(long requestId, String segment, long offset, String
         } else if (u instanceof BadKeyVersionException) {
             log.warn(requestId, "Conditional update on Table segment '{}' failed due to bad key version.", segment);
             invokeSafely(connection::send, new WireCommands.TableKeyBadVersion(requestId, segment, clientReplyStackTrace), failureHandler);
+        } else if (errorCodeExists(u)) {
+            log.warn(requestId, "Operation on Table segment '{}' failed due to a {}.", segment, u.getClass());
```

Comment:
This may not necessarily be on a `TableSegment`. I'd remove it (leave the rest).